mopidy_mpd/protocol/reflection.py

```diff
@@ -81,25 +80,6 @@ def notcommands(context):
     return [("command", command_name) for command_name in sorted(command_names)]
 
 
-@protocol.commands.add("tagtypes")
-def tagtypes(context, *parameters):
-    """
-    *musicpd.org, reflection section:*
-
-        ``tagtypes [ALL | CLEAR | DISABLE {NAME...] | ENABLE {NAME...]]``
-
-        Shows a list of available song metadata.
-    """
-    parameters = list(parameters)
-    if parameters:
-        subcommand = parameters.pop(0).lower()
-        if subcommand not in ("all", "clear", "disable", "enable"):
-            raise exceptions.MpdArgError("Unknown sub command")
-        # TODO: this is only a stub; we always return all tags anyhow.
-        return
-    return [("tagtype", tagtype) for tagtype in tagtype_list.TAGTYPE_LIST]
-
-
```

Comment:
why did you move this from reflection to connection? (not per se against it, but an explanation would be nice)

Reply:
The tagtypes command was listed under the connections settings section here: https://mpd.readthedocs.io/en/latest/protocol.html#connection. I moved it to be consistent with that.

Reply:
oh, you're right!